app/controllers/facilities_controller.rb

```diff
@@ -4,7 +4,7 @@ class FacilitiesController < ApplicationController
   # GET /facilities
   # GET /facilities.json
   def index
-    @facilities = Facility.all
+    @facilities = Facility.where(organization_id: current_user.organization_id)
```

Comment:
How do we feel about always scoping facility to the current user's organization?

I know some people haaaaaaaaaaate default scopes, but I wonder if it wouldn't be a good idea here.

@bhaibel what do you think?